WORKSPACE

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+workspace(name = "com_google_bazel_rules_hdl")
```

Comment:
Should this be something like `com_github_hdl_bazel_rules_hdl`?

Reply:
As per private chat, I'm changing it to `rules_hdl`